src/commands/cloud-login.js

```diff
@@ -1,134 +1,294 @@
-    'For login with 3rd-party identity providers like Github or Netlify, please acquire a key from Home > [database] > Security and enter it below instead.'
-  )
-  log('')
-
-  const credential = await cli.prompt('Email or secret key')
-  const strategy =
-    strategies[isEmail(credential) ? EMAIL_STRATEGY : SECRET_STRATEGY]
-
-  strategy(credential).catch(function (err) {
-    if (err.statusCode === 401) {
-      errorOut(JSON.parse(err.error).message, 1)
-    } else {
-      throw err
+  askAlias() {
+    return inquirer
+      .prompt([
+        {
+          name: 'alias',
+          message: 'The endpoint alias prefix. Would be combined with a region',
+          type: 'input',
+          default: this.environment.defaultAlias,
+
+          validate: (endpoint) =>
+            endpoint ? true : 'Provide an alias, please',
+        },
+        {
+          name: 'overwrite',
+          message: 'The endpoint alias already exists. Overwrite?',
+          type: 'confirm',
+          when: ({ alias }) => Boolean(this.config[alias]),
+        },
+      ])
+      .then((resp) => {
+        if (resp.hasOwnProperty('overwrite') && !resp.overwrite) {
+          this.askAlias()
+        } else {
+          this.alias = resp.alias
+        }
+      })
+  }
+
+  askAuth() {
+    return inquirer
+      .prompt([
+        {
+          name: 'auth',
+          message: 'Select an authorization flow',
+          type: 'list',
+          choices: [
+            { name: 'Email and Password', value: 'password' },
+            { name: 'Secret', value: 'secret' },
+            // { name: 'GitHub', value: 'github' },
+            // { name: 'Netlify', value: 'netlify' },
+          ],
+        },
+      ])
+      .then(({ auth }) => {
+        this.auth = auth
+      })
+  }
+
+  async askIsDefault(endpoints) {
+    if (!this.config.default && endpoints.length === 1) {
+      await setDefaultEndpoint(endpoints[0])
+      return this.log(`Endpoint '${endpoints[0]}' added as default`)
     }
-  })
-}
 
-function secretStrategy(secret) {
-  return saveEndpointOrError(newEndpoint, alias, secret)
-}
+    if (this.config.default === endpoints[0] && endpoints.length === 1) {
+      return this.log(`Endpoint '${endpoints[0]}' added`)
+    }
+
+    // If 1 new endpoint which is not a default one (and default exists), ask a user to consider it as default
+    // If more than 1 endpoints, ask which one a user would like to be a default (or keep existing)
 
-async function emailStrategy(email) {
-  const password = await cli.prompt('Password', {
-    type: 'hide',
-    timeout: 120000,
-  })
+    const { defaultEndpoint } = await inquirer.prompt([
+      {
+        name: 'setDefault',
+        message: `Would you like endpoint '${endpoints[0]}' to be default?`,
+        type: 'confirm',
+        when: endpoints.length === 1,
+      },
+      {
+        name: 'defaultEndpoint',
+        message:
+          'Endpoints created. Would you like to set some of them as default?',
```

Comment:
```suggestion
          'Endpoints created. Would you like to set one of them as default?',
```
There cannot be multiple defaults.